lib/config/index.js

```diff
@@ -42,6 +48,60 @@ const getParser = () => {
                         ? thr('Each browser must have "commands" option')
                         : assertArrayOfStrings(value, 'commands');
                 }
+            }),
+            nativeElementsSize: section({
+                topToolbarHeight: option({
+                    defaultValue: defaults.nativeElementsSize.topToolbarHeight,
+                    parseEnv: JSON.parse,
+                    parseCli: JSON.parse,
+                    validate: (value, _config, currentNode) => {
+                        if (
+                            (!_.isNull(value) && !_.isNumber(value)) ||
+                            (_.isNull(value) && !isNativeElementsSizeEmpty(currentNode))
```

Comment:
if one of the values is declared (for example `bottomToolbarHeight`) then in this case we will fall here with error `"topToolbarHeight" must be a number but got ...`.

Why not just declare `nativeElementsSize` as `option` (without `section`) and check each option inside `validate`? You can look at this example - https://github.com/gemini-testing/hermione/blob/master/src/config/browser-options.js#L160-L173

Reply:
Ok, `nativeElementsSize` declaration is updated